app/components/preprint-status-banner/component.js

```diff
@@ -223,15 +226,16 @@ export default Component.extend({
         return this.get('reviewerComment').trim() !== this.get('initialReviewerComment');
     }),
 
-    decisionChanged: computed('submission.reviewsState', 'decision', function() {
-        return this.get('submission.reviewsState') !== this.get('decision');
+    decisionChanged: computed('decision', 'initialDecision', function() {
+        return this.get('initialDecision') !== this.get('decision');
+    }),
+
+    pendingStateNotChanged: computed('submission.reviewsState', 'decision', function() {
```

Comment:
Is this property necessary? With `initialDecision` it seems like `decisionChanged`/`userActivity` are all you need.

Reply:
There is a special case where submission status is pending, however, user decision is 'accept' on the feedback form. I used the pendingStateNotChanged property to check for it.

Reply:
In that case, `initialDecision` and `decision` would both be `ACCEPTED`, so `decisionChanged` would be false, which is correct. Do we have to treat that case specially?

Reply:
Remember our discussion regarding the case where a user might change her/his decision (for pending preprints), then receives the warning, if the user restored decision to accept --> still should continue recieve the warning.

Reply:
Ah, right. That doesn't need to be a special case, though -- you could set `userActivity = true` whenever the user clicks on a decision or enters text in the comment box. That might let you simplify some of the logic in this component, which is good because it looks like it's impossible to accept a pending preprint:

![noaccept](https://user-images.githubusercontent.com/6776190/34786204-70e06a18-f601-11e7-878b-24d205654685.gif)